azkaban-web-server/src/main/java/azkaban/webapp/servlet/ProjectManagerServlet.java

```diff
@@ -1584,99 +1579,110 @@ private void handleJobPage(final HttpServletRequest req, final HttpServletRespon
   private void handlePropertyPage(final HttpServletRequest req,
       final HttpServletResponse resp, final Session session) throws ServletException {
     final Page page =
-        newPage(req, resp, session,
-            "azkaban/webapp/servlet/velocity/propertypage.vm");
+        newPage(req, resp, session, "azkaban/webapp/servlet/velocity/propertypage.vm");
     final String projectName = getParam(req, "project");
-    final String flowName = getParam(req, "flow");
+
+    // flow and job parameters are mainly used to build the breadcrumb.
+    final String flowNodePath = getParam(req, "flow");
     final String jobName = getParam(req, "job");
-    final String propSource = getParam(req, "prop");
+
+    // The name of the file where properties are located
+    final String propsSource = getParam(req, "prop");
```

Comment:
Is there any particular reason that we renamed this? Just curious.

Reply:
nope, I guess I assumed that there are multiple properties.